Packs/CommonScripts/Scripts/Jmespath/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```

Comment:
No need for this file.
Please add release notes according to the new convention.
https://xsoar.pan.dev/docs/integrations/changelog

Reply:
Just FYI, the demisto-sdk creates this file by default when initilising a new pack. Should it be there?

Reply:
No, its an sdk bug that should be resolved. 
btw don't forget to run the generate-release-notes command.
I don't see the new rn